flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroDeserializationSchema.java

```diff
@@ -128,6 +128,9 @@ Decoder getDecoder() {
 
     @Override
     public T deserialize(byte[] message) throws IOException {
```

Comment:
Please add annotation @NULLABLE before parameter message.

You can also take a look at the constructor of the KafkaDynamicSource, which allows its input is null.